src/app/ui/alpha_entry.cpp

```diff
@@ -0,0 +1,256 @@
+// Aseprite UI Library
+// Copyright (C) 2024  Igara Studio S.A.
+//
+// This file is released under the terms of the MIT license.
+// Read LICENSE.txt for more information.
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include "app/pref/preferences.h"
+#include "app/ui/alpha_entry.h"
+#include "base/scoped_value.h"
+#include "gfx/rect.h"
+#include "gfx/region.h"
+#include "os/font.h"
+#include "ui/fit_bounds.h"
+#include "ui/manager.h"
+#include "ui/message.h"
+#include "ui/popup_window.h"
+#include "ui/scale.h"
+#include "ui/size_hint_event.h"
+#include "ui/slider.h"
+#include "ui/system.h"
+#include "ui/theme.h"
+
+#include <algorithm>
+#include <cmath>
+
+namespace app {
+
+using namespace gfx;
+
+AlphaEntry::AlphaEntry(AlphaSlider::Type type)
```

Comment:
I think this class has too much in common with IntEntry, isn't a way to derive IntEntry and avoid duplicating code?